pdata/internal/generated_ptrace.go

```diff
@@ -1049,11 +1035,10 @@ func (es SpanLinkSlice) AppendEmpty() SpanLink {
 // can be compared.
 //
 // Returns the same instance to allow nicer code like:
-//
-//	lessFunc := func(a, b SpanLink) bool {
-//	  return a.Name() < b.Name() // choose any comparison here
-//	}
-//	assert.EqualValues(t, expected.Sort(lessFunc), actual.Sort(lessFunc))
+//   lessFunc := func(a, b SpanLink) bool {
+//     return a.Name() < b.Name() // choose any comparison here
+//   }
+//   assert.EqualValues(t, expected.Sort(lessFunc), actual.Sort(lessFunc))
```

Comment:
Hm.. it just replaces tabs with spaces in the comments. The template uses tabs. I don't think this is an improvement. Maybe is there something changed to how `go fmt` work in your setup?

Reply:
It's the output when I run `make genpdata` on main, but let me refresh my tools install and see what happens.

Reply:
Is old go fmt that you have @TylerHelmuth